frontend/src/apps/main/scenes/contact/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./style.scss";
+import ContactForm from "./Form";
```

Comment:
```suggestion
import ContactForm from "./form";
```
Again, filenames renaming needed